src/wp-includes/block-template-utils.php

```diff
@@ -1296,7 +1345,25 @@ function get_block_template( $id, $template_type = 'wp_template' ) {
 		return null;
 	}
 	list( $theme, $slug ) = $parts;
-	$wp_query_args        = array(
+
+	$active_templates = get_option( 'active_templates', array() );
+
+	if ( ! empty( $active_templates[ $slug ] ) ) {
+		if ( is_int( $active_templates[ $slug ] ) ) {
+			$post = get_post( $active_templates[ $slug ] );
+			if ( $post && 'publish' === $post->post_status ) {
+				$template = _build_block_template_result_from_post( $post );
+
+				if ( ! is_wp_error( $template ) && $theme === $template->theme ) {
+					return $template;
+				}
+			}
+		} elseif ( false === $active_templates[ $slug ] ) {
+			return null;
+		}
```

Comment:
Is this nested correctly? We can only get here via the ` ! empty( $active_templates[ #slug ] )` on L1351 (maybe we want an isset there? )